src/coreclr/inc/yieldprocessornormalized.h

```diff
@@ -3,14 +3,12 @@
 
 #pragma once
 
-// Undefine YieldProcessor to encourage using the normalized versions below instead. System_YieldProcessor() can be used where
-// the intention is to use the system-default implementation of YieldProcessor().
 #define HAS_SYSTEM_YIELDPROCESSOR
```

Comment:
I think `HAS_SYSTEM_YIELDPROCESSOR` can be deleted from here and everywhere else.